.github/workflows/tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: yarn jest --clearCache
 
       - name: Test
-        run: yarn test --coverage
+        run: yarn test

```

Comment:
@GerasNyx We must provide --coverage flag to generate coverage report for the pipeline